src/utils/constants.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace constants {
+
+// Usage strings
+constexpr const auto *kUsage =
+    "Memgraph bolt client.\n"
+    "The client can be run in interactive or non-interactive mode.\n";
+
+constexpr const auto *kInteractiveUsage =
+    "In interactive mode, user can enter cypher queries and supported "
+    "commands.\n\n"
+    "Cypher queries can span through multiple lines and conclude with a\n"
+    "semi-colon (;). Each query is executed in the database and the results\n"
+    "are printed out.\n\n"
+    "The following interactive commands are supported:\n\n"
+    "\t:help\t Print out usage for interactive mode\n"
+    "\t:quit\t Exit the shell\n";
+
+static const std::string kCommandQuit = ":quit";
+static const std::string kCommandHelp = ":help";
+
+// Supported formats.
+static const std::string kCsvFormat = "csv";
+static const std::string kTabularFormat = "tabular";
+
+// History default directory.
+static const std::string kDefaultHistoryBaseDir = "~";
+static const std::string kDefaultHistoryMemgraphDir = ".memgraph";
+// History filename.
+static const std::string kHistoryFilename = "client_history";
+
+static const std::string kPrompt = "memgraph> ";
+static const std::string kMultilinePrompt = "       -> ";
+
+/// Memgraph and OpenCypher keywords.
+//  ToDo(the-joksim):
+//    - add all the missing keywords
+static const std::vector<std::string> kMemgraphKeywords{
```

Comment:
Actually this can be `std::array`.

Reply:
Then it would break the API, as some util functions call e.g. `size` on the elements, and the compiler sees this as `std::array<const char *, whatever-the-size-is>`